style.css

```diff
@@ -0,0 +1,932 @@
+@import url("https://fonts.googleapis.com/css2?family=Gothic+A1:wght@100;200;300;400;500;600;700;800;900&family=Lato:wght@100;300;400;700;900&family=Roboto:wght@900&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: "Gothic A1", sans-serif;
+  font-family: "Lato", sans-serif;
+  font-family: "Roboto", sans-serif;
+  list-style: none;
+  text-decoration: none;
+  scroll-behavior: smooth;
+}
+
+body {
+  font-style: normal;
+  color: #333;
+  line-height: 1.6;
+  background: #f4f5f7;
+}
+
+:root {
+  --color-top-header: #3e3e3e;
+  --color-primary: #6c63ff;
+  --color-success: #00bf8e;
+  --color-first-nav: #f07f7e;
+  --color-showcase: #f8f8f8;
+  --color-hash: #595959;
+  --color-danger: #f75842;
+  --color-date: #868686;
+  --color-anchor-tag: #ff5438;
+  --color-parah: #515151;
+  --color-feature: #f78c89;
+  --color-danger-variant: #rgba(247, 88, 66, 0.4);
+  --color-white: #fff;
+  --color-nav-bg: whitesmoke;
+  --color-light: rgba(255, 255, 255, 0.7);
+  --color-black: #18191a;
+  --color-bg: #1f1f38;
+  --color-bg-variant: #2c2c6c;
+  --color-primary-variant: rgba(77, 181, 255, 0.4);
+}
+
+.container {
+  max-width: 1100px;
+  margin: 0 auto;
+  padding: 0 20px;
+  overflow: auto;
+}
+
+a {
+  color: var(--color-top-header);
+}
+
+p {
+  margin: 10px 0;
+}
+
+img {
+  width: 100%;
+}
+
+.nav-logo {
+  display: none;
+}
+
+.navbar {
+  height: 50px;
+  position: fixed;
+  z-index: 1;
+  width: 100%;
+}
+
+.navbar .container {
+  display: flex;
+  align-items: center;
+  height: 100%;
+  justify-content: start;
+}
+
+.navbar .nav-links {
+  display: none;
+}
+
+.navbar .open {
+  font-size: 30px;
+  opacity: 1;
+}
+
+.navbar .close {
+  font-size: 30px;
+  color: var(--color-anchor-tag);
+  align-items: end;
+}
+
+.navbar .mobile-nav {
+  font-family: 'Lato', sans-serif;
+  font-size: 25px;
+  display: none;
+  flex-direction: column;
+  position: fixed;
+  gap: 30px;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  background: var(--color-white);
+  z-index: 20;
+  width: 80%;
+  padding: 40px;
+  transition: 0.5s;
+}
+
+.card {
+  background-color: var(--color-white);
+  border-radius: 24px;
+  box-shadow: 0 3px rgba(0, 0, 0, 0.2);
+  padding: 20px;
+  margin: 10px;
+}
+
+.black {
+  color: var(--color-black);
+}
+
+.gray {
+  color: var(--color-date);
+}
+
+.top-header {
+  display: none;
+}
+
+.flex {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+}
+
+.btn {
+  display: inline-block;
+  padding: 10px 10px;
+  cursor: pointer;
+  color: var(--color-anchor-tag);
+  border: none;
+}
+
+.nav-links .btn-default {
+  background-color: transparent;
+  border: 5px solid var(--color-anchor-tag);
+}
+
+.navbar .btn-default a {
+  font-family: "Lato", sans-serif;
+}
+
+/* .navbar .nav-links  {
+  color: var(--color-hash);
+  font-family: 'Lato', sans-serif;
+  font-size: 17px;
+  font-weight: 200;
+} */
```

Comment:
- [ ] Please remove all unwanted commented code from your codebase because its make your codebase messy if you want to check your previous code then check GitHub commit history :slightly_smiling_face: